smelly_python/generator/webpage_generator.py

```diff
@@ -96,52 +114,50 @@ def generate_webpage(report, output_path=path.join('report', 'smelly_python')):
-                        file = smell.location.path
-                        html_path = html_paths[file]
-
-                        row = tr(_class='center-text')
-                        table_data = td()
-                        if smell.type == 'error':
-                            table_data.add(image(src='error.svg', alt='error'))
-                        elif smell.type == 'warning':
-                            table_data.add(image(src='warning.svg', alt='warning'))
-                        else:
-                            # Will be "refactor" or "convention"
-                            table_data.add(image(src='info.svg', alt='info'))
-
-                        row += table_data
-                        row += td(a(file, href=html_path))
-                        row += td(smell.symbol)
-                        row += td(smell.message)
-                        code_smell_link = f'{html_path}#code-block.{smell.location.line}'
-                        row += td(a(f'{smell.location.line}:{smell.location.column}',
-                                    href=code_smell_link))
+
+        if report.is_clean():
+            p('There were no code smells found. Good job!')
+            # raw('There were no code smells found! <strong>Good job!</strong>')
+
+        else:
+            with div():
+                with table():
+                    with thead():
+                        row = tr()
+                        row += th('Severity')
+                        row += th('File')
+                        row += th('Code smell')
+                        row += th('Message')
+                        row += th('Location')
+                    with tbody():
+                        for smell in report.code_smells:
+                            file = smell.location.path
+                            html_path = html_paths[file]
+
+                            row = tr(_class='center-text')
+                            table_data = td()
+                            if smell.type == 'error':
+                                table_data.add(image(src='error.svg', alt='error'))
+                            elif smell.type == 'warning':
+                                table_data.add(image(src='warning.svg', alt='warning'))
+                            else:
+                                # Will be "refactor" or "convention"
+                                table_data.add(image(src='info.svg', alt='info'))
+
+                            row += table_data
+                            row += td(a(file, href=html_path))
+                            row += td(smell.symbol)
+                            row += td(smell.message)
+                            code_smell_link = f'{html_path}#code-block.{smell.location.line}'
+                            row += td(a(f'{smell.location.line}:{smell.location.column}',
+                                        href=code_smell_link))
 
         with footer():
             raw('<strong>Icons by svgrepo.com</strong>')
```

Comment:
If there are no code smells, we're not showing any icons.
Should this still be on there then? (maybe because we are still using them, but I'm not sure)
I guess when we place the footer at the bottom of the page it won't matter that much anymore, but still.

Reply:
Yeeted it